docs/tutorials/mcmc_global.ipynb

```diff
@@ -220,26 +220,14 @@
   },
```

Comment:
<p>I think it would be useful to have some text here about the fact that there are default luminosity functions, but also how to use your own.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/21cmfast/21CMMC/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>